tidb/assets/configuration/spec.yaml

```
@@ -0,0 +1,63 @@
+name: TiDB
+files:
+  - name: tidb.yaml
+    options:
+      - template: init_config
+        options:
+          - template: init_config/openmetrics_legacy
+      - template: instances
+        options:
+          - template: instances/openmetrics_legacy
+            overrides:
+              prometheus_url.value.example: http://localhost:2379/metrics
+              namespace.value.example: pd
```

Since the check reach different ports of an endpoint, I think a way to do that easily would the the following:
```suggestion
          - name: base_url
             description: "The base URL of the different tidb endpoints"
             required: true
             value:
               type: string
               example: "http://localhost"
          - template: instances/openmetrics_legacy
            overrides:
              prometheus_url.hidden: true
              namespace.hidden: true
```
And then the `prometheus_url` of all the instances can be built as `base_url:{port}`